integration/constants.go

```diff
@@ -8,7 +8,8 @@ const (
 	StartPortSimulationInMem         = 22000
 	StartPortSimulationFullNetwork   = 26000
 	StartPortSmartContractTests      = 30000
-	StartPortContractDeployerTest    = 34000
+	StartPortContractDeployerTest1   = 34000
+	StartPortContractDeployerTest2   = 35000
```

Comment:
@anixon604  spotted this issue. We used the same port in 2 test in the same file. Would fail if tests run in parallel